### PR TITLE
Sentry Integration added

### PR DESCRIPTION
**Pull request by deepak1725**
Description

Added Sentry Logs integration.
The developer now can use an instance to log errors into Sentry.
Added Decorator as logging helper

Fixes # (issue)
Type of change

Please delete options that are not relevant.
[x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/recommendation_service.py [hunk 1]
```diff
@@ -86,5 +92,13 @@ def recommendation():
     return flask.jsonify(response_json), 200
 
 
+def log_it(func):
```
Comment (miteshvp) [inline src/recommendation_service.py]: Where is this decorator function used?

Comment (deepak1725) [reply src/recommendation_service.py]: future reference.

Comment (miteshvp) [reply src/recommendation_service.py]: What future reference are you planning?

Comment (deepak1725) [reply src/recommendation_service.py]: sir, i have an idea of wrapping any new function with this decorator. 
if this doesn't make sense.. i am okay to drop this

Comment (miteshvp) [reply src/recommendation_service.py]: Your idea is good. Maybe we should include this at the time of implementing this. But not with this PR.

Comment (miteshvp) [reply src/recommendation_service.py]: Please change accordingly in all other PR's as well.

Comment (deepak1725) [reply src/recommendation_service.py]: okay

**Pull request merged**